articles/dms/tutorial-mysql-azure-mysql-offline-portal.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "Tutorial: Migrate MySQL offline to Azure Database for MySQL"
```

Comment:
Should we update https://docs.microsoft.com/en-us/azure/dms/resource-scenario-status so that is shows offline support for MySQL is in preview?

Reply:
Done